atd-vze/src/containers/DefaultLayout/DefaultHeader.js

```diff
@@ -98,6 +98,12 @@ const DefaultHeader = props => {
               >
                 <i class="fa fa-wrench" /> Request an enhancement&nbsp;&nbsp; <i className="fa fa-external-link" />
               </DropdownItem>
+              <DropdownItem
+                href="https://ftp.dot.state.tx.us/pub/txdot-info/trf/crash_notifications/cr3_code_sheet.pdf"
+                target="_blank"
```

Comment:
I remembered seeing a new cheat sheet in the april release, and started looking through the url folder that this one was in and found this link
https://ftp.dot.state.tx.us/pub/txdot-info/trf/crash_notifications/2023/code-sheet.pdf

Reply:
Yes, awesome. I updated the link to point to this most recent cheat sheet. Also, sent a quick note asking him if that was okay. He thanked us for using the most updated version.